src/core/output/outputGenerate.ts

```
@@ -5,55 +5,91 @@ import { RepopackError } from '../../shared/errorHandle.js';
+
+  return outputs;
 };
 
 export const buildOutputGeneratorContext = async (
   rootDir: string,
   config: RepopackConfigMerged,
-  allFilePaths: string[],
-  processedFiles: ProcessedFile[],
+  includedFiles: ProcessedFile[] = [], // Add includedFiles parameter
+  repositoryStructure: string[] = [],
+  totalFiles: number = 1,
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)